controlpanel/api/models/tool.py

```diff
@@ -95,6 +96,7 @@ def save(self, *args, **kwargs):
         """
         Deploy the tool to the cluster (asynchronous)
         """
+        self._id_token = kwargs.get('id_token')
```

Comment:
Maybe I'm missing something: where is `kwargs` defined here?

Reply:
Sorry, that was GitHub collapsed the method definition and it looked like this was in the body of the constructor 😆